app/controllers/admin/problems_controller.rb

```diff
@@ -1,4 +1,64 @@
 # frozen_string_literal: true
 
 class Admin::ProblemsController < Admin::ApplicationController
+  before_action :set_session
+  before_action :set_mission
+  before_action :set_problem, only: %i[edit show update]
+
+  def index
+    problems = Problem.where('mission_id = ?', @mission.id).order('name')
+    @q = problems.ransack(params[:q])
+    @problems = @q.result&.page(params[:page] || 1)&.per(10)
+  end
+
+  def new
+    @problem = @mission.problems.new
+  end
+
+  def create
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Metrics/AbcSize: Assignment Branch Condition size for create is too high. [16.61/15]